src/aqdata/tests/lsgc_test.cc

```diff
@@ -40,7 +40,7 @@ void LSGCTest::AQDataTest() {
   lsgc_ptr->MakeAQ();
   auto wi = lsgc_ptr->GetAQWeights();
   auto omega_i = lsgc_ptr->GetAQDirs();
-  for (unsigned int i=0; i<wi.size(); ++i)
+  for (int i=0; i<wi.size(); ++i)
```

Comment:
It occurs to me, after our discussion of `int` vs `unsigned`, we should consider using `std::size_t i = 0` for these loops.

Reply:
Ah interesting! I should have checked the style guide, that works, sorry!

For vectors, at least, `std::vector::size_type` _is_ just a typedef of `size_t`:
```
class vector :
...
      typedef size_t					size_type;
...
```

Reply:
Thanks for the good question!

I will look at it later. If that's true, I don't understand about the compiler warning when trying `size_t`. But anyways, if we stick to Google Style Guide, we'll use `int` for sure.